packages/scene-composer/src/store/helpers/serializationHelpers.ts

```diff
@@ -127,14 +142,18 @@ function createTagComponent(
 
   validateRuleBasedMapId(ruleBasedMapId, resolver, errorCollector);
 
+  let dataBindingContext: any;
```

Comment:
You shouldn't be making any changes in createTagComponent this is rebase/merge issue.